media/js/firefox/releasenotes.js

```diff
@@ -4,23 +4,11 @@
 
 /* eslint no-unused-vars: [2, { "varsIgnorePattern": "stickyNav" }] */
 
-(function($, Mozilla, Waypoint) {
+(function($, Mozilla) {
     'use strict';
 
     var client = Mozilla.Client;
     var $html = $('html');
-    var $nav = $('#nav');
-
-    var stickyNav = new Waypoint.Sticky({
-        element: $nav,
-        handler: function(direction) {
-            if (direction === 'down'){
-                $nav.addClass('fixedNav');
```

Comment:
It looks like there are some [css rules](https://github.com/mozilla/bedrock/blob/master/media/css/firefox/releasenotes-firefox.less#L108-L118) that could be deleted with the removal of this. 🙂

Reply:
✅